packages/frontend/src/deviceMessages.ts

```diff
@@ -26,16 +26,13 @@ export async function updateDeviceChat(
     }
   }
 
-  await addDeviceMessage('changelog-version-1.50.0', {
-    text: `What's new in 1.50.0?
+  await addDeviceMessage('changelog-version-1.52.0', {
+    text: `What's new in 1.52.0?
 
-🎹 Improved keyboard navigation
-📤 Select and send multiple files at once
-🎮 Enhanced in-chat apps: Get notifications and open supporting apps in context, i.e. open an added calendar entry directly
-🚀 In-chat apps can be really fast now, thanks to the "realtime" support
-📋 Add private tags for accounts
+    🚀 New App Picker available in attachment menu! Shows apps from https://webxdc.org
+    🎹 Improved keyboard navigation
 
-MORE ✨ improvements and 🐜 bug fixes see [Full Changelog](https://github.com/deltachat/deltachat-desktop/blob/main/CHANGELOG.md#1_50_0)`,
+MORE ✨ improvements and 🐜 bug fixes see [Full Changelog](https://github.com/deltachat/deltachat-desktop/blob/main/CHANGELOG.md#1_52_0)`,
```

Comment:
Is this how we do it? Isn't it gonna send another message which is the same as the one we sent for 1.50?

Reply:
Oh. Never mind, I'm stupid. I thought this is the same message.